__tests__/controller_functions.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const { string } = require("pg-format");
+const { checkIfReviewIdExists, isIdValid } = require('../controllers/controller_functions');
+
+
+describe('checkIfReviewIdExists', () => {
```

Comment:
You don't need to perform this, if there is no article your database will tell you by returning an empty array.